__tests__/test-statusMatrix-in-submodule.js

```
@@ -404,4 +404,28 @@ describe('statusMatrix', () => {
       ['b/b.txt', 0, 2, 2],
     ])
   })
+
+  it('does not modify .git/index when refresh is false', async () => {
+    // Setup
+    const { fs, dir, gitdir } = await makeFixtureAsSubmodule(
+      'test-statusMatrix'
+    )
```

ESLint

[error] Replace `·await·makeFixtureAsSubmodule(⏎······'test-statusMatrix'⏎····` with `⏎······await·makeFixtureAsSubmodule('test-statusMatrix'`

(prettier/prettier)